resources/i18n/it_IT/cura.po

```diff
@@ -2901,19 +2686,19 @@ msgstr "Apri file progetto"
 
 msgctxt "@title:window Don't translate 'Universal Cura Project'"
 msgid "Open Universal Cura Project (UCP)"
-msgstr ""
+msgstr "Aprire il progetto universale Cura (UCP)"
```

Comment:
Here too, good translation, but you claim it to be untranslated

Reply:
Yeah apparently no single translator saw the warning, so it ends up being translated in all langauges :smiling_face_with_tear: